### PR TITLE
[Frontend] Fixed table horizontal scrolling issue

### PR DESCRIPTION
**Pull request by frankcbw**
Fixed horizontal scrolling issue in advanced-filter-table, where table headers and table body were not aligned correctly when scrolling.

### CODE REVIEW

**Comment by frankcbw** [comment]
Unused lines of code are commented out in case of revision needed later. 
Will be removed when the PR is ready to merge.

**Comment by Accelsnow** [comment]
Looks good! This should automatically resolve the scrolling issue in the delete ddah dialog and offer withdraw dialog, as they also use the AdvaceFilterTable implementations

**Comment by siefkenj** [comment]
Remove the comments and remove `react-custom-scrollbars` from `package.json` if it is no longer needed.

File: frontend/src/components/filter-table/advanced-filter-table.tsx [hunk 1]
```diff
@@ -186,27 +174,19 @@ export function AdvancedFilterTable({
                 <HiddenRowWarning visible={isHiddenRowsSelected} />
                 <div {...table.getTableBodyProps()} className="tbody">
                     <AutoSizer>
-                        {({ width, height }) => {
+                        {({ height }) => {
                             // Don't let the table get too short no matter what
                             //height = Math.max(height, 300);
```
Comment (siefkenj) [inline frontend/src/components/filter-table/advanced-filter-table.tsx]: Why is this commented out? If it's not needed, it should be removed.

**Pull request merged**